packages/x-adapter-platform/src/types/requests/index.ts

```diff
@@ -1,3 +1,4 @@
-export * from './popular-searches-request.model';
-export * from './query-suggestions-request.model';
 export * from './identifier-results-request.model';
+export * from './query-suggestions-request.model';
+export * from './popular-searches-request.model';
+export * from './search-request.model';
```

Comment:
Next queries is missing here but I think that is solved on main already.

Reply:
Updated with main and now Next queries are there.